### ISSUE TITLE
Custom Trades with diverse professions still broken in 1.08

### ISSUE THREAD

**Issue by crux-f** [GC-spigot/AdvancedTrades#11]
### Describe the bug

This is a follow up to issue #7 in which all custom trades were broken.

The bug was reportedly fixed in 1.08, however the error still occurs.

See issue #7 for details such as logs and screenshots.

### How to reproduce

The error found in #7 has not been fixed; If there is more than one profession assigned to any two trades, the entire custom trade event breaks. For example if you have two custom trades and both of them are assigned to Armorer, the plugins works as expected. But if one trade is assigned to Butcher and the other to Armorer, Paper throws the ‘unable to pass trade event’ error in the console, and stops functioning.

**Comment by crux-f** [comment]
@GC-spigot Good afternoon,
Are there any updates on when this bug will be fully resolved? I am attempting to launch a server utilizing its features in under 3 weeks, and currently have no way of using it. 

Let me know when you can,

Thank you

**Comment by crux-f** [comment]
@GC-spigot Good afternoon,

There are now only two weeks left until my server is scheduled to launch, and this plugin still does not work.

While I understand there have been personal challenges for you, the lead developer, I have gathered through conversations on the discord that none of the other developers are able to work on this plugin.

This seems in and of itself like a red flag for any continued support of this plugin, and I’m half inclined to abandon this entirely, as I have needed to continuously bother everyone I can get in touch with to even get a chance at using the product I paid for.

I’m willing to give you and your team the benefit of the doubt, but at this point the lack of communication is really frustrating me, especially considering how many completely free plugins there are out there with much better communication.

In conclusion, I hope that I can get the product I paid for at some point within the next two weeks, and if not I will be attempting to program a solution myself.
